### ISSUE TITLE
Front End Report With Stripe

### ISSUE THREAD

**Issue by elanasparkle** [BoldGrid/weforms#117]
Can use Frontend report if payment is processed first
=========================================================
Name           |   Front End Stripe
Reporter       |   Elana D.
Submit Date |   7/20/2021
Summary      | When the frontend report is enabled then stripe payment processed the form doesn't display and produces the below error. 

![image](https://user-images.githubusercontent.com/5271694/126315565-8d366ead-ed5c-4316-a5cc-9a5b7bc24c18.png)

**Comment by avonville** [comment]
@elanasparkle We don't really want to show frontend reports for payments? Is this something that a customer is asking for? what would be the use case where we would want to show payment data to anyone that submits the form?

A possible feature would be to make a donation report that said how much has been donated if that is something customers would use.

**Comment by elanasparkle** [comment]
@avonville This isn't a front-end report for payments. This is the front-end report option in the settings. 

![image](https://user-images.githubusercontent.com/5271694/141119700-66c662f3-d929-4955-b375-4893c86b1fa3.png)


To reproduce: Enable front-end reports in the settings then add a payment method on the form. Payments successful or failed it doesn't redirect to the frontend report afterward.

**Comment by avonville** [comment]
Yes I know what the frontend report does, it shows the entry data for that form. So for instance if it did work...

![image](https://user-images.githubusercontent.com/25781560/141807794-312942b1-aafa-41bf-8dee-e1d0017272c3.png)

It would show a report of all the entries with this data. What I am asking is what is the use case for them having a frontend report when using that form for payment.

**Comment by avonville** [comment]
We should loop harry and tony in on this and get their thought to see how we want to move forward.

**Comment by harryjackson1221** [comment]
Your screenshot shows a payment and amounts, and that front end report should show the information you have in your screenshot. The current issue is that if stripe is used, said report is not working. Can you elaborate on what you are saying please? @avonville

**Comment by avonville** [comment]
So the payment is the last piece that happens in the submission process. So the front-end report is running before the payment redirect is done. Furthermore, stripe checkout redirects and dies to a specific URL that is set. So if we were to have the frontend report run it would have go within the payments addon when a payment is enabled.

I still don't see a use case that has been brought up to where a customer is going to want people to see the payment information of all individuals that have submitted the form.

**Comment by avonville** [comment]
In addition to what I said in the previous comment, I believe that the frontend report option should be disabled when a payment processor is enabled.

**Comment by harryjackson1221** [comment]
It should redirect to the report after payment is made. If we want to add the payment info, we can do that later but for right now the error should not show when payment info is enabled. @avonville
